kurtosis-module/impl/module_io/default_params.go

```diff
@@ -62,9 +71,9 @@ func GetDefaultExecuteParams() *ExecuteParams {
 		},
 		WaitForMining:       true,
 		WaitForFinalization: false,
-		ClientLogLevel:      ParticipantLogLevel_Info,
+		ClientLogLevel:      unspecifiedGlobalParticipanLogLevel,
```

Comment:
Realistically we should probably change this to be `GlobalClientLogLevel` and rename the enum accordingly

Reply:
Also typo (missing `t`)